tests/integration_tests/modules/test_users_groups.py

```diff
@@ -86,3 +89,30 @@ def test_user_root_in_secret(self, class_client):
         _, groups_str = output.split(":", maxsplit=1)
         groups = groups_str.split()
         assert "secret" in groups
+
+
+@pytest.mark.user_data(USER_DATA)
+def test_sudoers_includedir(client: IntegrationInstance):
+    """Ensure we don't add additional #includedir to sudoers.
+
+    Newer verisons of /etc/sudoers will use @includedir rather than
+    #includedir. Ensure we handle that properly and don't include an
+    additional #includedir when one isn't warranted.
+
+    https://github.com/canonical/cloud-init/pull/783
+    """
+    if ImageSpecification.from_os_image().release not in ['hirsute', 'groovy']:
```

Comment:
This looks like it'll require updating in ~2mos; should we do `in ['xenial', 'bionic', 'focal']` instead? (Or perhaps `@pytest.mark.not_xenial`, `.not_bionic`, and `.not_focal`?)

Reply:
Yep, good idea. I prefer the former these days because otherwise we'll have 8-10 marks specifying every current release and not-release (plus future releases), and that's just for ubuntu. In the back of my mind, I have an idea for having a single mark to specify OS/feature dependencies...but...that doesn't help us now :smile:

Reply:
Yeah, I was just thinking that `@pytest.mark.ubuntu(not=["xenial", "bionic", "focal"])` or similar would work nicely for this.